src/Connection.php

```diff
@@ -328,7 +328,7 @@ private static function lookupVersion(): string
     {
         if (class_exists(InstalledVersions::class)) {
             try {
-                return self::$version = InstalledVersions::getPrettyVersion('jenssegers/laravel-mongodb');
+                return self::$version = InstalledVersions::getPrettyVersion('jenssegers/mongodb');
```

Comment:
I suggest don't use assignment in return block, and separate operation assignment and return value

Reply:
Updated the code accordingly 👍